api/v1beta1/azuremanagedcontrolplane_webhook_test.go

```diff
@@ -2559,6 +2559,29 @@ func TestAzureManagedControlPlane_ValidateUpdate(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "AzureManagedControlPlane DNSPrefix is immutable ok nil with cluster name",
```

Comment:
```suggestion
			name: "AzureManagedControlPlane DNSPrefix is immutable or nil with cluster name",
```

Reply:
I agree this name is bad as-is but I might tweak it slightly from your suggestion.